ExperimentConfigSidecar/Models/SubscriptionSpec.cs

```diff
@@ -0,0 +1,58 @@
+namespace ExperimentConfigSidecar.Models;
+
+/// <summary>
+/// Dapr subscription configuration.
+/// </summary>
+public class SubscriptionSpec
+{
+    /// <summary>
+    /// The topic to subscribe to.
+    /// </summary>
+    public string Topic { get; set; }
+
+    /// <summary>
+    /// The pubsub name.
+    /// </summary>
+    public string Pubsubname { get; set; }
+
+    /// <summary>
+    /// The metadata.
+    /// </summary>
+    public Dictionary<string, string> Metadata { get; set; }
+
+    /// <summary>
+    /// The route.
+    /// </summary>
+    public string Route { get; set; }
+
+    /// <summary>
+    /// Bulk subscription configuration.
+    /// </summary>
+    public BulkSubscribe BulkSubscribe { get; set; }
+
+    /// <summary>
+    /// The max number of retries.
+    /// </summary>
+    public string? DeadLetterTopic { get; set; }
+}
+
+/// <summary>
+/// Dapr bulk subscription configuration.
+/// </summary>
+public class BulkSubscribe
+{
+    /// <summary>
+    /// The bulk subscription configuration.
```

Comment:
Is this a correct summary?
How does the bool describe a bulk subscription configuration

Reply:
changed with 13fb59a8872aa2042202e017d1fd30d1b29144be